modules/dateAndTime.js

```diff
@@ -0,0 +1,9 @@
+function updateTime(timeP) {
+  const today = new Date();
+  const date = `${today.getFullYear()}-${today.getMonth() + 1}-${today.getDate()}`;
+  const time = `${today.getHours()}:${today.getMinutes()}:${today.getSeconds()}`;
+  const dateTime = `${date} ${time}`;
+  timeP.innerHTML = dateTime;
```

Comment:
- [ ] Please 🙏🏻 adhere to the project requirement according to which you should use the `luxon` library for displaying the `Date And Time`, but it seems like you created your custom functionality, which is very good and I really appreciate ❤️  it, but as a developer when you will do work in a company then at that time you would strongly follow the guidelines and requirements and so that's why **Microverse** made a well structured document for the projects to follow the things properly. Kindy use the `luxon` library which you already have installed and added into your `package.json`. 

### Requirement
![Screenshot from 2023-07-10 16-35-50](https://github.com/agneta1991/Awesome-books-with-ES6/assets/66476329/117efc95-ce4f-4269-b6f3-a71be2198e04)